default_executor/src/lib.rs

```diff
@@ -19,11 +19,10 @@ static ALLOC: LockedAllocator<FreeListAllocator> = LockedAllocator::new(FreeList
 /// # Warning
 ///
 /// The defaults are not guaranteed to be stable.
-#[derive(Clone, Constructor, Debug, ValidateEntrypoints, ExpressionEvaluator, Validate, Visit)]
+#[derive(Clone, Constructor, Debug, ValidateEntrypoints, Validate, Visit)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Constructor, Visit, Validate, ValidateEntrypoints)]
```

Also, unrelated. I dislike seeing `ValidateEntrypoints`. Seems out of place

Reply:
> Also, unrelated. I dislike seeing ValidateEntrypoints. Seems out of place

What would you prefer instead?

Reply:
- Created #4123